azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -1976,6 +1994,19 @@ private void lockFlowsForProject(final Project project, final List<String> locke
     }
   }
 
+  private void setNeedsMoreUsersIfAppropriate(final Page page, final Project project, final Session session) {
+    // See if the project has only one admin user with no admin groups.
+    // If it does AND the forceMultipleUsers flag is set AND the current user has the ability to add
+    // an admin, let's force the user to do so before allowing any execution or scheduling of flows.
+
+    boolean hasEnoughUsersGroups = project.getUsersWithPermission(Type.ADMIN).size() > 1
+                              || project.getGroupsWithPermission(Type.ADMIN).size() > 0
+                              || !hasPermission(session.getUser(), Type.ADMIN)
+                              || !this.forceMultipleUsers;
```

Comment:
Move this.forceMultipleUsers to the front, if the config is off, we fallback to existing logic.

Reply:
😄